### DISCUSSION TITLE
contributing: cannot run docs site

### DISCUSSION THREAD

**Discussion by thediveo** [mdx-js/mdx#1398] [Q&A] [answered]
I would like to work on a PR for the documentation regarding create-react-app.

- forked mdx-js/mdx and cloned my fork; as of commit 2529cd73435672b2e94e221370d21bfdf863907e
- `git branch`: `main`
- https://github.com/mdx-js/mdx#getting-started: `npm init mdx`
- https://mdxjs.com/contributing#running-the-docs-site...
  - `yarn` ... ok
  - `yarn build` ... ok
  - `yarn docs`
    - ERROR #11329 ... whatever that means?
    - ERROR Cannot set property 'esSyntax' of undefined in mdx-loader.js:79:25

What am I doing wrong?

**Comment by wooorm** [comment, marked as answer]
Thanks for working on the docs!

You're not doing anything wrong. We're right in the middle between v1 and v2. The website for v2, from 'main', seems to currently be broken. Whereas the 'master' branch (for v1) probably still works (I think?)

So you might try and work on 'master' at the moment, however, as it's roughly a markdown file, perhaps you don't need to build the website itself to contribute docs?

**Reply by thediveo** [reply]
I'm now on branch `master` and can successfully start run `yarn docs`; however, when loading the page in Chromium (Version 87.0.4280.88 (Official Build) snap (64-bit) on Ubuntu 20.10), it crashes:

![image](https://user-images.githubusercontent.com/6920158/102696556-996ebe80-422f-11eb-9a94-3d7d9024920b.png)
